app/src/main/java/com/example/tanya/godeltechchallengeandroid/di/module/AppModule.kt

```diff
@@ -4,15 +4,12 @@ import android.app.Application
 import android.content.Context
 import android.content.res.Resources
 import com.example.tanya.godeltechchallengeandroid.App
-import com.example.tanya.godeltechchallengeandroid.api.ApiModule
-import com.example.tanya.godeltechchallengeandroid.data.prefs.ApplicationRepository
-import com.example.tanya.godeltechchallengeandroid.data.prefs.ApplicationRepositoryImpl
 import dagger.Binds
 import dagger.Module
 import dagger.Provides
 
 
-@Module(includes = [ApiModule::class])
+@Module
 abstract class AppModule {
```

Comment:
The same here too. You can simplify your code. Please see my comment related to the `APiModule` class.